### PR TITLE
Update to version 0.2.0

### PR DESCRIPTION
**Pull request by kylacochrane**
Update `gasnomenclature` to version 0.2.0.

See `CHANGELOG.md` for updates.

### CODE REVIEW

**Review by emarinier — APPROVED** [review]

File: assets/samplesheet.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 sample,mlst_alleles,address
-sampleQ,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sampleQ.mlst.json,
-sample1,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sample1.mlst.json,1.1.1
-sample2,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sample2.mlst.json,1.1.1
-sample3,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sample3.mlst.json,1.1.2
+sampleQ,https://raw.githubusercontent.com/phac-nml/gasnomenclature/update/input_assure/tests/data/reports/sampleQ.mlst.json,
```
Comment (apetkau) [inline assets/samplesheet.csv:2]: Can you update the branch here for test data back to `dev`?

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks great Kyla. Just one small change.

File: assets/samplesheet.csv (hunk 1 above)
Comment (kylacochrane) [reply assets/samplesheet.csv]: Thank you for the find! Changed here: [03b0d3a](https://github.com/phac-nml/gasnomenclature/pull/21/commits/03b0d3a4a2257ec32dbaf69ab4588bf4fe9c3c2d)

**Review by apetkau — APPROVED** [review]
Thanks so much Kyla. Looks great 😄

**Pull request merged**